provider/azure/config_test.go

```diff
@@ -42,7 +42,14 @@ func (s *configSuite) TestValidateNew(c *gc.C) {
 func (s *configSuite) TestValidateInvalidStorageAccountType(c *gc.C) {
 	s.assertConfigInvalid(
 		c, testing.Attrs{"storage-account-type": "savings"},
-		`invalid storage account type "savings", expected one of: \["Standard_LRS" "Standard_GRS" "Standard_RAGRS" "Standard_ZRS" "Premium_LRS"\]`,
+		`invalid storage account type "savings", expected one of: \["Premium_LRS" "Premium_ZRS" "Standard_GRS" "Standard_LRS" "Standard_RAGRS" "Standard_ZRS"\]`,
+	)
+}
+
+func (s *configSuite) TestValidateInvalidLoadBalancerSkuName(c *gc.C) {
+	s.assertConfigInvalid(
+		c, testing.Attrs{"load-balancer-sku-name": "premium"},
+		`invalid load balancer SKU name "Premium", expected one of: \["Basic" "Standard"\]`,
```

Comment:
does the case matter to the Azure end? Can we be more forgiving of case here?

Reply:
We are. As you can see from the error message, we're transforming the input from "premium" to "Premium" on the user's behalf.